common/logger/logger.ts

```diff
@@ -10,7 +10,10 @@ addLayout('json', function () {
         const error = logEvent.context['error'] || {};
         delete logEvent.context['error'];
 
-        return JSON.stringify({ ...logEvent, message, data, error });
+        // These tags will be used to identify the logs in datadog later on
+        const tags = { env: process.env.ENV, version: process.env.VERSION || 'latest' };
+
+        return JSON.stringify({ ...logEvent, message, data, error, tags, service: process.env.SERVICE_NAME });
```

Comment:
@Jonasdoubleyou could you please set a `SERVICE_NAME` environment variable for the api/worker dynos?

Maybe something like
```
SERVICE_NAME=backend-api
SERVICE_NAME=backend-worker
```

Reply:
I turned on [Dyno Metadata](https://devcenter.heroku.com/articles/dyno-metadata) so you could use HEROKU_DYNO_ID to distinguish them. Unfortunately we cannot set the ENV per Dyno otherwise. 

Alternatively we could set the service name in package.json, i.e.
```
"web": "SERVICE_NAME=backend-api node ...",
```